docs/secondary-setup.md

```diff
@@ -0,0 +1,6 @@
+## Secondary setup
```

Comment:
This is a detail about how the custom WBStack code works. It could make sense to put this in a docs folder inside the `dist-persist` directory.

Reply:
Putting any docs inside `dist-persist` make the Github Actions failed, so I moved it back to `/docs`